Instruction/Instruction_text.txt

```diff
@@ -23,12 +23,12 @@ So, for example, when you rolled three twos and one four and one five, you could
 If the upper section sums up to 63 or more points, you will get a bonus of 35 points to the total score.
 
 Lower section
-•	Three of a kind: this category may only be filled if the dice show at least three of the same numbers (5, 5, 5, 1, 3). The score for this category is the total of the dice
-•	Four of a kind: score of the total of all dice, only when you have 4 dices with the same number
-•	Full house: you need to have 3 dice with the same number and 2 dice with the same number (for example: 2, 2, 3, 3, 3 or 1, 1, 4, 4, 4). A full house is worth 25 points
-•	Small Straight: any sequence of four numbers. Note it can also be five consecutive numbers. A small straight will give you 30 points.
-•	Large Straight: a sequence of five numbers, for example: 1,2,3,4,5 or 2,3,4,5,6. This is valued 40 points
-•	Chance: This offers a player the opportunity to score on any turn where he does not choose to score in any of the other open boxes. Mark score by totalling number of points on all dice.
-•	Yahtzee: five of a kind. Yahtzee is the highest possible score (50 points).
+** Three of a kind: this category may only be filled if the dice show at least three of the same numbers (5, 5, 5, 1, 3). The score for this category is the total of the dice
+** Four of a kind: score of the total of all dice, only when you have 4 dices with the same number
+** Full house: you need to have 3 dice with the same number and 2 dice with the same number (for example: 2, 2, 3, 3, 3 or 1, 1, 4, 4, 4). A full house is worth 25 points
+** Small Straight: any sequence of four numbers. Note it can also be five consecutive numbers. A small straight will give you 30 points.
+** Large Straight: a sequence of five numbers, for example: 1,2,3,4,5 or 2,3,4,5,6. This is valued 40 points
+** Chance: This offers a player the opportunity to score on any turn where he does not choose to score in any of the other open boxes. Mark score by totalling number of points on all dice.
+** Yahtzee: five of a kind. Yahtzee is the highest possible score (50 points).
 The player with the highest score wins!
 
```

Comment:
hi dana, here we go again :) my brain is already a bit fried, so I am trying my best now, but if you feel like youre still stuck somewhere or just want an opinion on something, let me know !!!!

Reply:
The instructions seem otherwise very clear to me.